clippy_lints/src/formatting.rs

```diff
@@ -10,7 +10,7 @@ use rustc_span::source_map::Span;
 
 declare_clippy_lint! {
     /// ### What it does
-    /// Checks for usage of the non-existent `=*`, `=!` and `=-`
+    /// Checks for usage of the nonexistent `=*`, `=!` and `=-`
```

Comment:
Both the hyphenated and non-hyphenated versions are used. No need to remove the change, but the spell checker is overzealous.

Reply:
That's actually me, the human. I have some preferences. My main preference is for consistency (it's much easier to search and get all matches than search and get <1/2 of matches - as there''s often a distribution between two accepted spellings and some misspellings of the same), and my secondary preference is to not hyphenate.

I'm certainly willing to drop if people have opinions.